transcom/mymove
# B-19723-INT Office viewers can view EMPLID

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19723)

## Summary

Part of the effort to incorporate EMPLID into the app for USCG customers. Office users should now see the EMPLID in the header for office users, as well as the Move Details Customer Info section, if the customer is USCG

### How to test

1. Create a new customer, branch USCG and enter an emplid
2. Submit a move
3. Login to the office application and find the move - confirm that all places where the DODID is displayed, the EMPLID is as well
4. Confirm that non USCG customers do not have EMPLID displayed anywhere
